src/main/java/compsci4039/week1/loops/Lab4Ex2.java

```diff
@@ -62,13 +64,15 @@ private static void executeGuessingGame() {
         int numberOfAttempts = 10;
         int currentAttemptIndex = 0;
 
-        System.out.println("Hello! Welcome to my awesome guessing game.\nGuess which number I've picked. You can choose between from 0 - 99");
+        System.out.println(
+                "Hello! Welcome to my awesome guessing game.\nGuess which number I've picked. You can choose between from 0 - 99");
```

Comment:
We should either commit the formatting rules (e.g. the ```.prettierrc``` file) or never stage those to the addition. No problem for now as we are unlikely to further develop this class :)

Reply:
Good plan! Might be worth working on some boilerplate environment stuff including a .prettierrc (I'm pretty sure it has a java plugin) as well as a gradle build for unit tests, jdoc, etc for future assignments... Might try and make one and push it to my github. Maybe we could convince some of the more experienced coders to dabble in devops...

Gradle seems like the way to go after comparing it with Maven (XML...yuck.).  

Might as well over-engineer these labs 😛